### PR TITLE
Simplified urls.py

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: haystack/urls.py [hunk 1]
```diff
@@ -1,16 +1,12 @@
 # encoding: utf-8
 
-from __future__ import absolute_import, division, print_function, unicode_literals
```
Comment (acdha) [inline haystack/urls.py]: This doesn't seem like a necessary change. I'd like to leave that in so we have a consistent __future__ policy across every file.

Comment (claudep) [inline haystack/urls.py]: Yes, I've seen that afterwards. I might suggest a cleaning patch later, as some of these imports are no longer necessary with Python 2.7.

File: test_haystack/core/urls.py [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
+from __future__ import unicode_literals
 
-from django.conf.urls import include, patterns, url
+from django.conf.urls import include, url
 from django.contrib import admin
 
+from haystack import views
 from haystack.forms import FacetedSearchForm
 from haystack.query import SearchQuerySet
-from haystack.views import FacetedSearchView, SearchView
```
Comment (acdha) [inline test_haystack/core/urls.py]: I would prefer to revert this change because it allows static linters to verify that referenced variables exist without having to load views.py

File: test_haystack/core/urls.py [hunk 3]
```diff
@@ -1,28 +1,26 @@
...
 
 
-urlpatterns = patterns('',
-    (r'^admin/', include(admin.site.urls)),
-)
+urlpatterns = [
+    url(r'^admin/', include(admin.site.urls)),
 
+    url(r'^$', views.SearchView(load_all=False), name='haystack_search'),
+    url(r'^faceted/$', views.FacetedSearchView(searchqueryset=SearchQuerySet().facet('author'),
```
Comment (acdha) [inline test_haystack/core/urls.py]: This should probably be indented to match the `FacetedSearchView(` so it's visually obvious which parameters are being passed to which function.

**Comment by acdha** [comment]
Other than the very minor things I added comments for, this looks good to me — there's plenty of room for this now that we don't have to support anything newer than Django 1.8.
